internal/ask/form/aggregation.go

```diff
@@ -299,53 +298,32 @@ func TextAggregate(context interface{}, db *db.DB, formID string, subs []submiss
 				continue
 			}
 
-			// Get the answer and options.
-			a, ok := ans.Answer.(map[string]interface{})
-			if !ok {
-				return nil, fmt.Errorf("Could not parse answer")
-			}
-
+			// Options == nil points to a non MultipleChoice answer.
 			var answer string
-			options, ok := a["options"]
-			if !ok || options == nil {
-
+			a := ans.Answer.(bson.M)
```

Comment:
Why is not checking to see if it is the right cast? During testing we found that it was failing to convert into bson.M.

It should be:

a, ok := ans.Answer.(bson.M)

and check on ok!